miri

```diff
@@ -48,6 +48,25 @@ Pass extra flags to all cargo invocations.
 EOF
 )
 
+## Run the subcommands that the user requested to always run first
+if [ -z "$AUTO_OPS" ]; then
+    export AUTO_OPS=42
+
+    # Run this first, so that the toolchain doesn't change after
+    # other code has run.
+    if [ -f ".auto-everything" ] || [ -f ".auto-toolchain" ] ; then
+        "$MIRIDIR"/rustup-toolchain
```

Comment:
That was too far up. :joy: now MIRIDIR is not defined yet.

Reply:
Huh... I tested it and it seemed to work. Maybe bash just makes unknown env vars be empty?

Reply:
> Maybe bash just makes unknown env vars be empty?

Yes it definitely does that. That is "expected" behavior in shell script land. ;)